### PR TITLE
hoc2015: some more work on levels & logic.

### PR DESCRIPTION
**Pull request by breville**

### CODE REVIEW

**Comment by breville** [comment]
@bcjordan Here's the skeleton of 15 levels.  All asset names are generic (character/background/item/destination/etc.), and assets themselves are reused from previous tutorials.

File: apps/src/studio/Item.js [hunk 1]
```diff
@@ -71,8 +70,8 @@ Item.prototype.createElement = function (parentElement) {
   this.element.setAttributeNS('http://www.w3.org/1999/xlink', 'xlink:href',
     this.image);
   this.element.setAttribute('id', itemId);
-  this.element.setAttribute('height', this.height * this.animationFrames);
-  this.element.setAttribute('width', this.width * this.frames);
+  this.element.setAttribute('height', this.height * this.frames);
+  this.element.setAttribute('width', this.width * 8);
```
Comment (bcjordan) [inline apps/src/studio/Item.js]: What does the `8` represent? Maybe introduce named constant for this

File: apps/src/studio/collidable.js [hunk 2]
```diff
@@ -186,9 +198,12 @@ Collidable.prototype.roamGrid = function(type) {
           candidate.score ++;
         } else if (type == "chaseGrid") {
           if (candidateY == this.gridY - 1 && spriteY < this.y - bufferDistance) {
-            candidate.score ++;
+            candidate.score += 2;
           } else if (candidateY == this.gridY + 1 && spriteY > this.y + bufferDistance) {
-            candidate.score ++;
+            candidate.score += 2;
+          }
```
Comment (bcjordan) [inline apps/src/studio/collidable.js]: Join w/ next line

File: apps/src/studio/collidable.js [hunk 3]
```diff
@@ -186,9 +198,12 @@ Collidable.prototype.roamGrid = function(type) {
           candidate.score ++;
         } else if (type == "chaseGrid") {
           if (candidateY == this.gridY - 1 && spriteY < this.y - bufferDistance) {
-            candidate.score ++;
+            candidate.score += 2;
```
Comment (bcjordan) [inline apps/src/studio/collidable.js]: Maybe introduce constant for this & next usage if they should match up

File: apps/src/studio/studio.js [hunk 4]
```diff
@@ -3611,6 +3687,10 @@ Studio.getPlayspaceBoundaries = function(sprite)
   return boundaries;
 };
 
+Studio.getSkin = function()
```
Comment (bcjordan) [inline apps/src/studio/studio.js]: join with next line

**Comment by bcjordan** [comment]
LGTM! Not sure, but may need to make change in last comment to avoid linter complaint

**Pull request merged**